src/lib/wallet.js

```diff
@@ -55,13 +59,55 @@ async function getKeyPair() {
 }
 
 async function initContractInstances() {
-  if (!store.getters.mainnet && !process.env.RUNNING_IN_TESTS) return;
+  if (!store.getters.mainnet && !process.env.RUNNING_IN_TESTS && process.env.NETWORK !== 'Testnet')
+    return;
   const contractAddress = await store.dispatch('getTipContractAddress');
-  store.commit(
-    'setTipping',
-    await store.state.sdk.getContractInstance(TIPPING_CONTRACT, {
-      contractAddress,
-      forceCodeCheck: true,
+  const contractAddressV2 = await store.dispatch('getTipContractAddressV2');
+  const contractInstance = await store.state.sdk.getContractInstance(TIPPING_CONTRACT, {
+    contractAddress,
+    forceCodeCheck: true,
+  });
+  const contractInstanceV2 = await store.state.sdk.getContractInstance(TIPPING_CONTRACT_V2, {
+    contractAddress: contractAddressV2,
+    forceCodeCheck: true,
+  });
+  store.commit('setTipping', contractInstance);
```

Comment:
why is V1 still required?

Reply:
We are about to use it upon implementing "retip" functionality with the deeplinks. If the user would pick an older post to retip.

Reply:
maybe it should be explicitly named V1 then :)

Reply:
This would require further refactoring. I propose to optimize it into a separate refactoring PR.